TRACK_README.md

```diff
@@ -1,11 +1,58 @@
 # Exercism {{LANGUAGE}} Track
 
+![build status](https://travis-ci.org/exercism/xslug.svg?branch=master)
+
 Exercism exercises in {{LANGUAGE}}.
 
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading the general Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
```

Comment:
We've got a bit of a mess with respect to contributing guides. That one is now just about the problems API.

Should we link to https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md (It's a bit of a wall of text).

Or perhaps [getting involved in a track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md)?